DigiMeSDK/Postbox/Classes/DMEClient+Postbox.m

```diff
@@ -0,0 +1,84 @@
+//
+//  DMEClient+Postbox.m
+//  DigiMeSDK
+//
+//  Created by Jacob King on 16/10/2018.
```

Comment:
Naughty naughty - please remove your name and add copyright statement to this and header file